_about/who-we-are.html

```diff
@@ -18,8 +18,9 @@
 </div>
 
 <div class="biobox">
-  <p><b>J.D. Zamfirescu</b><br>
-  J.D. runs Workshop Weekend, a community-based educational program similar to an all-ages Splash, in Oakland, California. He helped to found Stanford ESP in 2008 and co-founded AppJet, a web collaboration startup, in 2007. AppJet produced a web-based real-time collaborative text editor, EtherPad, which was acquired by Google in 2009. Before moving to San Francisco, he directed programs at MIT ESP and spent a year as a software engineer at Google in New York. J.D. holds both bachelor's and master's degrees in computer science from MIT.</p>
+  <div class="imagebox floatleft"><img src='/media/images/biopic/vanessa_bw.jpg" width="143" height ="200" alt="Vanessa"></div>
+  <p><b>Vanessa Barksdale (Chair)</b><br>

```

Comment:
Should probably remove above where it says I'm chair, and also format this like the other titles.